pkg/certificate/triple/cert.go

```diff
@@ -80,6 +90,13 @@ func NewSelfSignedCACert(cfg Config, key crypto.Signer, duration time.Duration)
 		BasicConstraintsValid: true,
 		IsCA:                  true,
 	}
+
+	// Golang < 1.15 is not filling in the SKI field for CAs [1] so we replicate the golang >= 1.15 fix [2]
```

Comment:
Are we okay with overwriting `tml.SubjectKeyId` when we move to golang >= 1.15? Shouldn't we check `tmpl.SubjectKeyId` is empty first?

Reply:
That was my original idea but it means that it will call CreateCertificate twice at golang < 1.15 another option is to check the golang version but that's even worse and the third option is to do nothing and just try to bump to 1.15 all the parts.

Reply:
ahh, yeah gotcha.